bigtable/tests/unit/test_instance.py

```diff
@@ -330,7 +325,7 @@ def test_create_w_explicit_serve_nodes(self):
         client._instance_admin_client.bigtable_instance_admin_stub = stub
 
         # Perform the method and check the result.
-        result = instance.create()
+        result = instance.create(location_id=self.LOCATION_ID)
```

Comment:
Hmm, the name of the testcase suggests that we should be passing all the non-default arguments to `Instance.create` here (and asserting they are propagated correctly).

Reply:
I definitely agree that this needs to be changed.  I added [this comment](https://github.com/GoogleCloudPlatform/google-cloud-python/issues/5454#issuecomment-400371391) to issue #5454 to track this. This PR was intended just to be an initial refactoring, and I hope that it's OK to keep this PR as such.

There's quite a bit more work required to complete issue #5454, so is it ok to improve the test cases as a follow up PR?